pinot-spi/src/main/java/org/apache/pinot/spi/utils/ControllerRequestURLBuilder.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.controller.helix;
+package org.apache.pinot.spi.utils;
```

Comment:
Move it under `org.apache.pinot.spi.utils.builder`

Reply:
Ack